modules/nodes-nodes-audit-config-about.adoc

```diff
@@ -22,6 +22,15 @@ Audit log profiles define how to log requests that come to the OpenShift API ser
 
 |`AllRequestBodies`
 |In addition to logging metadata for all requests, logs request bodies for  every read and write request to the API servers (`get`, `list`, `create`, `update`, `patch`). This profile has the most resource overhead. ^[1]^
+
+|`None`
+|No requests are logged; even OAuth access token requests and OAuth authorize token requests are not logged.
+
+[WARNING]
+====
+It is not recommended to disable audit logging by using the `None` profile unless you are fully aware of the risks of not logging data that can be beneficial when troubleshooting issues. If you disable audit logging and a support situation arises, you might need to enable audit logging and reproduce the issue in order to troubleshoot properly.
```

Comment:
@bergerhoffer Why say the same `WARNING` four times？ I thought we just emphasize it here once.

Reply:
@wangke19 It might seem a little overkill, but I put it once in each file that mentions the None profile. This is because someone could potentially jump right to one of the sections instead of reading an earlier "about" section. And technically the file could get reused elsewhere in our doc set without the context that has the warning.

I think that the warning is important enough (since there are support risks if you choose to use it), that I'd rather have it appear too often rather than get missed.

I'll run it by another doc team member though as well to see whether they think it's too much or not. @kalexand-rh would you mind taking a look and giving me your thoughts on this?

Reply:
@wangke19 I agree with Andrea. Using the None profile sounds like a major support risk. While we'd like for users to read the entire rendered assembly to get the entire workflow, every module has to make sense when read alone. I wouldn't want to risk a user not having this information when they needed it, and the note seems like crucial information everywhere it shows up.

Reply:
OK. 
lgtm